test_files/sickle/declare.ts

```diff
@@ -0,0 +1,18 @@
+declare module DeclareTest {
+  // This enum should not cause any extra sickle statements to be
+  // emitted, because it is contained within a "declare" block.
+  enum FileType {
+    VALID = 1,
+  }
+}
+
+module DeclareTestIncluded {/** @typedef {number} */
+
+  // This enum should produce statements, as it's not a "declare".
+  enum FileType {
+    VALID = 1,
+  }
+/** @type {FileType} */
+(<any>FileType).VALID =  1;
```

Comment:
What this change is addressing is that if we emit this bit of the processed enum in the above module (with the "declare"), it is a TypeScript syntax error because you cannot include statements within a "declare" block.

I'm not sure this solution is great.  :\